src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -94,17 +88,44 @@ Each service has at least one settings file. These are expected to be named `set
 }
 ```
 
+## Functionality in the Builds (FR Look Here)
+The Secret Manager is set up to run 3 separate parts in our builds: usage validation during the testing phase, vault validation before deployment, and secret rotation on a weekly cadence.
+
+### Usage Validation
+This step runs the `validate-all` command. This command finds all `settings.json` and `settings.<environment>.json` files and validates those files with the manifests in the repo. This step doesn't talk to azure at all, it simply validates that all `[vault()]` references in the settings files have matching specifications in the appropriate manifest. Failures in this step require either adding missing definitions to the manifest, or removing the offending references in the settings file.
+
+### Pre-Deployment Validation
+This step runs the `synchronize --verify-only` command before the "approval" stage for the deployments. This command compares each manifest with the contents of the corresponding vault, and produces errors if secrets in the vault are missing or out-of-date. Failures in this step can be fixed by running the weekly build again, or by manually running the `synchronize` command.
+
+### Weekly Rotation
+This step runs the `synchronize` command. This compares every secret specified in the manifest with the corresponding vault, and performs the appropriate rotation for each secret that requires it. Failures in this step are caused by secrets that require rotation and can't be rotated manually. To manually rotate these secrets a human will need to run the `synchronize` command manually and specify the manifest that contains the secret requiring rotation.
```

Comment:
```suggestion
This step runs the `synchronize` command. This compares every secret specified in the manifest with the corresponding vault, and performs the appropriate rotation for each secret that requires it. Failures in this step are caused by secrets that require rotation and can't be rotated automatically. To manually rotate these secrets a human will need to run the `synchronize` command manually and specify the manifest that contains the secret requiring rotation.
```

Right? Otherwise... I'm missing something about how syncronize works.